CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+8.2.0...
+
+### Breaking changes
```

Comment:
is this where we define breaking change text?

Reply:
I'm not sure 🙈 @torkelo do you know?

Reply:
@achatterjee-grafana any thoughts on how to best do this?

I also added links, but not sure that is the right syntax :)

Reply:
I think this is the right place to announce breaking changes. See https://github.com/grafana/grafana/blob/main/CHANGELOG.md#breaking-changes-3.

Tagging @pkolyvas.

Reply:
Looks like the script will read from text in the issue summary -- I moved this text to that section.  Thanks!